app/components/CrashLocationBanner.tsx

```diff
@@ -15,10 +17,16 @@ export default function CrashLocationBanner({
 }: CrashLocationBannerProps) {
   return (
     <Alert variant="warning">
-      This crash is not included in Vision Zero statistical reporting because{" "}
-      {privateDriveFlag
-        ? "it occurred on a private drive"
-        : "it is located outside of the Austin full purpose jurisdiction"}
+      <AlignedLabel>
+        <FaTriangleExclamation className="me-2 d-none d-lg-inline" />
```

Comment:
i added icons to both of these banners. it gets really squished on smaller displays, so i added the responsive display class so that it is hidden below `lg`.